### ISSUE TITLE
Unable to input unicode characters in Windows

### ISSUE THREAD

**Issue by devster31** [zeit/hyper#3977]
- [x] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [x] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**: Windows 10.0.18362
- **Hyper.app version**: 3.0.2
- **Link of a [Gist](https://gist.github.com/) with the contents of your .hyper.js**: https://gist.github.com/devster31/e67574e2a0d3cf28d3aeb6438c956eb6
- **Relevant information from devtools** _(CMD+ALT+I on macOS, CTRL+SHIFT+I elsewhere)_: N/A
- **The issue is reproducible in vanilla Hyper.app**: this is straight out of the box Hyper.app

## Issue
I am currently trying to input the ratio character using PowerShell and being logged into an `ssh` session. I am using the default `ssh.exe` found on the version of Windows detailed above.
I am trying to input the character using AutoHotKey which seems to work outside of the ssh session.
To recap:

* PowerShell + ssh allows the input
* PowerShell within Hyper allows the input
* Powershell + ssh within Hyper does not seem to work

```
this is the character in question, I have not tried with other unicode characters
∶
```
